### ISSUE TITLE
Komga: Crashes the app when receives a non-successful response

### ISSUE THREAD

**Issue by SageeW** [tachiyomiorg/tachiyomi-extensions#12158]
### Source information

Komga 1.3.40

### Steps to reproduce

Whenever I want to download manga and hit the download button, it crashes. Though it goes to the download queue and I can resume downloading from there, whenever I want to download a new chapter or manga, it crashes Tachiyomij2k, which is really annoying. 

### Expected behavior

Download chapter without crashing TachiyomiJ2k

### Actual behavior

Crashing the app

### Tachiyomi version

1.5.4

### Android version

Android 11

### Other details

https://user-images.githubusercontent.com/107306320/173187319-28d8d96c-e685-461b-8d6d-15c7a1044d80.mp4

[tachiyomi_crash_logs.txt](https://github.com/tachiyomiorg/tachiyomi-extensions/files/8883837/tachiyomi_crash_logs.txt)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [x] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by AntsyLich** [comment]
Wrong repo. Go to https://github.com/Jays2Kings/tachiyomiJ2K

**Comment by alessandrojean** [comment]
The crash logs indicates you got an Unauthorized answer from your Komga instance, so maybe double-check your credentials in the extension settings first?

But indeed the extension should handle this particular case.

**Comment by SageeW** [comment]
That's strange because, well, the extension asks me to authorize every time I turn on the server. I reinstalled the extension, cleared cookies, cleared the download queue, and asked around in support, and other people were experiencing the same issue. Maybe something is wrong with that crash log? I'm not an expert, but maybe it would help to send it again.

Also, I did check my credentials and re-entered my ip and everything again, but it doesn't help.
